storage/raft_transport.go

```diff
@@ -133,7 +133,9 @@ func (t *RaftTransport) RaftMessage(stream MultiRaft_RaftMessageServer) (err err
 					t.mu.Unlock()
 
 					if !ok {
-						return errors.Errorf("Unable to proxy message to node: %d", req.Message.To)
+						return errors.Errorf(
+							"unable to proxy message to node %d: store %d not registered",
```

Comment:
eh, that's not right either. why are we logging a node anyway? we should log `req.ToReplica`.